src/utils/source-maps.js

```diff
@@ -11,10 +11,9 @@ import type { SourceLocation, MappedLocation, Source } from "../types";
 import { isGenerated } from "../utils/source";
 
 export async function getGeneratedLocation(
-  state: Object,
+  sourceMaps: Object,
```

Comment:
I think we should avoid passing state into lots of helpers... 

This is an opinionated refactor to prevent this.